applications/search/src/pages/search-page/filter-pills/filter-pills.component.jsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import localization from '../../../lib/localization';
+import { getTranslateText } from '../../../lib/translateText';
+import { Pill } from './pill/pill.component';
+
+export const FilterPills = ({
+  history,
+  location,
+  locationSearch,
+  themesItems,
+  publishers
+}) => {
+  if (!(locationSearch && Object.keys(locationSearch).length > 0)) {
+    return null;
+  }
+
+  const pills = (history, location, locationSearch, themesItems, publishers) =>
```

Comment:
funksjonen er for langt
- ekstrahere blokker som funksjoner
- bruk switch i stedet av if, hvis valg av flere varianter.

Reply:
ekstrahert.